content/plugins/moritzebeling/kirby-panel-button/plugin.txt

```diff
@@ -0,0 +1,18 @@
+Title: Panle Button
```

Comment:
Panle should be Panel

Reply:
Ah ja sorry and thanks. I added a new commit to fix it. Can you merge it or shall I open a new PR?
Thank you!